daemon/lokinet-vpn.cpp

```diff
@@ -225,6 +235,15 @@ main(int argc, char* argv[])
     }
     return 0;
   }
+
+  if (not options.swapExits.empty())
+  {
+    nlohmann::json opts{{"exit_addresses", std::move(options.swapExits)}};
+
+    if (not OMQ_Request(omq, connectionID, "llarp.swap_exits", std::move(opts)))
```

Comment:
we dont actually need a new endpoint on the other side for this.
we can just do two requests  where the second one doesn't fire unless the first one is successful.

Reply:
That would push complexity down into the caller which doesn't seem desirable: now the client has to have logic about the proper order and handling of the swap, whereas a `swap_exits` endpoint provides an atomic (from the caller's perspective) operation.